ocaml/xapi/xapi_pool.ml

```diff
@@ -3567,16 +3567,31 @@ let disable_repository_proxy ~__context ~self =
         Db.Secret.destroy ~__context ~self:old_secret_ref
     )
 
+let get_uefi_certificates ~__context ~self =
+  let custom_certs = Db.Pool.get_custom_uefi_certificates ~__context ~self in
+  match (!Xapi_globs.allow_custom_uefi_certs, custom_certs) with
+  | false, _ | true, "" ->
```

Comment:
This suggests the empty string has a special meaning. Is this obvious for the user?

Reply:
In the design clearing the custom UEFI certificates would default back to using the default ones.

Reply:
Some docs can be added clarifying this clearing. An alternative could be to add an api function called `Pool.clear_custom_uefi_certs`

Reply:
Note that this special meaning of an empty value is already present currently in `Db.Pool.{set,}uefi_certificates`. We just moved this to the new field, to keep the amount of changes to a minimum.

Reply:
I've added in the doc of the setter the defaulting back when empty behavior. :)